### ISSUE TITLE
Add me!

### ISSUE THREAD

**Issue by annaek** [github/training-kit#584]
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

#### Screenshots
<If bug fix, please upload images of the error to better showcase the problem.>

#### Reasoning
<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it!
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository.
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by hollenberry** [comment]
Hey @annaek ! 👋  I think you might be looking for this repository: https://github.com/githubschool/open-enrollment-classes-introduction-to-github/issues

We're looking forward to seeing you over there! 😄
